src/include/types/CollectionSchema.h

```diff
@@ -107,6 +107,20 @@ class CollectionSchema {
         return true;
     }
 
+    /**
+     * @brief Return Anns field name, if not exist return ""
+     */
+    std::string
+    AnnsFieldName() const {
+        for (const auto& field : fields_) {
+            auto data_type = field.FieldDataType();
+            if (data_type == DataType::BINARY_VECTOR || data_type == DataType::FLOAT_VECTOR) {
```

Comment:
Currently, each collection only supports one vector field, this function is no problem. But in later version(maybe 2.1), each collection can manage multiple vector fields. This function could be a potential bug in future.

Reply:
I'd like refactor to `std::unorder_set<std::string> AnnsFieldNames()` and use it for validation.

Reply:
Ok, return `std::unordered_set<std::string>` is reasonable.